### PR TITLE
[all] add contributors and license

### PR DESCRIPTION
**Pull request by hwchen**

### CODE REVIEW

**Comment by MarcioPorto** [comment]
@hwchen Looks good on my end! 👍

**Comment by jspeis** [comment]
sounds good 👍

File: LICENSE-MIT [hunk 1]
```diff
@@ -0,0 +1,25 @@
+Copyright (c) 2016 keyring Developers
```
Comment (jspeis) [inline LICENSE-MIT]: one tweak though the license file should probably replace "keyring" with "tesseract"?

Comment (jspeis) [reply LICENSE-MIT]: and year to 2019

Comment (hwchen) [reply LICENSE-MIT]: Thanks for the catch :)

Comment (MarcioPorto) [reply LICENSE-MIT]: I don't know if you can test it before you merge this, but I think if you name the file as something like `LICENSE.md` GitHub picks it up and shows in the project navigation on the main page.

**Pull request merged**